src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -119,7 +119,6 @@
     <Compile Include="System\Xml\Core\XmlSubtreeReader.cs" />
     <Compile Include="System\Xml\Core\XmlSubtreeReaderAsync.cs" />
     <Compile Include="System\Xml\Core\XmlTextEncoder.cs" />
-    <Compile Include="System\Xml\Core\BinaryCompatibility.cs" />
```

Comment:
(You can also convert System.Threading.Tasks.Extensions from project reference to contract reference like all the other ones while you are on it.)